manifests/rhsm_reconfigure_script.pp

```diff
@@ -0,0 +1,57 @@
+define foreman_proxy_content::rhsm_reconfigure_script($ca_cert, $server_ca_cert) {
```

Comment:
I'm wondering if this needs to be a define. We only use it once AFAIK so might as well be a class, right?

```suggestion
# @summary Generate the RHSM reconfigure script
# @api private
define foreman_proxy_content::rhsm_reconfigure_script($ca_cert, $server_ca_cert) {
```